components/tooltip/stories/tooltip.stories.js

```diff
@@ -162,10 +162,12 @@ const PlacementVariants = (args) => html`
 									content: [`${optionDescription()}`],
 								})}
 							`)}
+							<div>
 							${Template({
 								...args,
 								placement: option,
 							})}
+							</div>
```

Comment:
Wrapping the Template in a <div> solves the issue that was happening here where the Tooltip starts expanding to fill the container:
![image (1)](https://github.com/adobe/spectrum-css/assets/54716846/2229f007-39bf-40fd-b40f-bd0c8e73ebe5)